.travis.yml

```diff
@@ -128,6 +128,17 @@ script:
   - if [[ $RUN_EXAMPLES == 'True' ]]; then py.test examples/cfd/example_diffusion.py; fi
   - if [[ $RUN_EXAMPLES == 'True' ]]; then py.test examples/seismic/elastic/elastic_example.py; fi
   - if [[ $RUN_EXAMPLES == 'True' ]]; then py.test examples/seismic/viscoelastic/viscoelastic_example.py; fi
+
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py test -P tti -so 4 -d 20 20 20 -n 5; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py test -P acoustic; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py bench -P acoustic; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py bench -P elastic; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py bench -P tti; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py bench -P viscoelastic; fi
+  - if [[ $RUN_EXAMPLES == 'True' ]]; then python benchmarks/user/benchmark.py plot -P acoustic --flop-ceil 80 linpack --max-bw 12.8; fi
```

Comment:
these 4 tests only test that the execution doesn't fail in python. They don't test whether an image is produced, etc. So I'm not sure how useful are they

Reply:
Well this was my purpose for having a first naive coupling between devito and opescibench.